conjure-java-core/src/main/java/com/palantir/conjure/java/types/AliasGenerator.java

```diff
@@ -259,6 +250,55 @@ private static boolean isAliasOfDouble(AliasDefinition typeDef) {
                 && typeDef.getAlias().accept(TypeVisitor.PRIMITIVE).equals(PrimitiveType.DOUBLE);
     }
 
+    private static ParameterSpec getAliasFactoryParameter(
+            Type aliasType, TypeName aliasTypeName, TypeMapper typeMapper, Options options) {
+        ParameterSpec parameterSpec = Parameters.nonnullParameter(aliasTypeName, "value");
+
+        if (aliasType.accept(TypeVisitor.IS_SET)) {
+            AnnotationSpec deserializeAnnotation = AnnotationSpec.builder(JsonDeserialize.class)
```

Comment:
nit: might be worth calling out this change in the commit message? 

I think we've already settled on this but writing it down - Using `LinkedHashSet` by default now is fine: previously we didn't guarantee an ordering and now we do, so there's no wire break. According to the LinkedHashSet docstring, there is a slight performance degradation compared to HashSet from maintaining a doubly-linked list, but the benefits of providing ordering outweigh the perf cost.